src/client/utils.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2025, Salesforce, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { SourceComponent, SourceComponentWithContent } from '../resolve/sourceComponent';
+
+export const isWebAppBundle = (component: SourceComponent): component is SourceComponentWithContent =>
```

Comment:
was repeated in a few places.  made it a TS guard to help stuff downstream of the guard not have to barbarically assert `content!`